### PR TITLE
Quit Whitespaces in methods

### PR DESCRIPTION
**Pull request by yorch81**
Quit whitespaces in the 2 methods added.

Cheers everybody.

### CODE REVIEW

File: lib/Zyberspace/Telegram/Cli/Client.php [hunk 1]
```diff
@@ -372,4 +372,34 @@ public function getHistory($peer, $limit = null, $offset = null)
 
         return $this->exec('history ' . $this->escapePeer($peer) . $limit . $offset);
     }
+
```
Comment (zyberspace) [inline lib/Zyberspace/Telegram/Cli/Client.php:375]: Codacy Issue found: [Whitespace found at end of line](https://www.codacy.com/app/zyberspace/php-telegram-cli-client/file/1977414991/issues/source?bid=2302274&fileBranchId=3116167#l375)

File: lib/Zyberspace/Telegram/Cli/Client.php [hunk 2]
```diff
@@ -372,4 +372,34 @@ public function getHistory($peer, $limit = null, $offset = null)
 
         return $this->exec('history ' . $this->escapePeer($peer) . $limit . $offset);
     }
+   
+    /**
+     * Send Picture to peer
+     * 
+     * @param  string $peer  The peer, gets escaped with escapePeer()
+     * @param  string $pic   Picture Path
+     * @return boolean
```
Comment (zyberspace) [inline lib/Zyberspace/Telegram/Cli/Client.php:381]: Codacy Issue found: [Whitespace found at end of line](https://www.codacy.com/app/zyberspace/php-telegram-cli-client/file/1977414991/issues/source?bid=2302274&fileBranchId=3116167#l381)

File: lib/Zyberspace/Telegram/Cli/Client.php [hunk 3]
```diff
@@ -372,4 +372,34 @@ public function getHistory($peer, $limit = null, $offset = null)
 
         return $this->exec('history ' . $this->escapePeer($peer) . $limit . $offset);
     }
+   
+    /**
+     * Send Picture to peer
+     *
```
Comment (zyberspace) [inline lib/Zyberspace/Telegram/Cli/Client.php:378]: Codacy Issue found: [Whitespace found at end of line](https://www.codacy.com/app/zyberspace/php-telegram-cli-client/file/1977414991/issues/source?bid=2302274&fileBranchId=3116167#l378)

**Comment by zyberspace** [comment]
Hey i will close this pr as it seems to be the same as #23 and doesn't fix any whitespaces.
I will merge #23 and remove the whitespaces by myself. :)

**Comment by yorch81** [comment]
Yes thanks, I not found the error I am using Sublime Text 2 in Linux.

Cheers everybody.

2016-02-05 10:56 GMT-06:00 Eric Enold notifications@github.com:

**Comment by yorch81** [comment]
This is my class.

<?php
use \Zyberspace\Telegram\Cli\Client;

class MyTelegram extends \Zyberspace\Telegram\Cli\Client
{
public function __construct()
{
parent::__construct('unix:///tmp/tg.sck');

}

/**
     \* Send Picture to peer
     *
     \* @param  string $peer  The peer, gets escaped with escapePeer()
     \* @param  string $pic   Picture Path
     \* @return boolean
     */
    public function sendPicture($peer, $pic)
    {
        $peer = $this->escapePeer($peer);
        $picture = $this->formatFileName($pic);

```
    return $this->exec('send_photo ' . $peer . ' ' . $picture);
}

/**
 * Send File to peer
 *
 * @param  string $peer  The peer, gets escaped with escapePeer()
 * @param  string $pic   File Path
 * @return boolean
 */
public function sendFile($peer, $file)
{
    $peer = $this->escapePeer($peer);
    $atFile = $this->formatFileName($file);

    return $this->exec('send_file ' . $peer . ' ' . $atFile);
}
```

}

?>

2016-02-05 10:59 GMT-06:00 Jorge Alberto Ponce Turrubiates <
the.yorch@gmail.com>:

**Pull request closed without merging**